features/personas/ChangePassword.feature

```diff
@@ -0,0 +1,12 @@
+@javascript @changePassword
+Feature: Verify than an User allowed to change password can change his password
+
+  Background:
+    Given I open Login page
+    And I log in as "UserChange" with password "Passw0rd-42"
+
+  Scenario: I can change my password
+    Given I go to change my password
+    And I change password from "Passw0rd-42" to "Passw0rd-43"
+    When I click on the edit action bar button "Update"
+    Then success notification that "Your password has been successfully changed." appears
```

Comment:
Does it make sense to add a check that user is redirected to Dashboard after the password change? (I've tested on v3).

Reply:
It behaves the same way on v2.5. 
Added in https://github.com/ezsystems/ezplatform-admin-ui/pull/1764/commits/3093c904caf093b598c734609c7ec09467bdf492.